src/reanimated2/animation/util.ts

```diff
@@ -190,6 +202,105 @@ function decorateAnimation<T extends AnimationObject | StyleLayoutAnimation>(
     return finished;
   };
 
+  // START
+
+  const transformationMatrixOnStart = (
+    animation: Animation<AnimationObject>,
+    value: AffiniteMatrixFlat,
+    timestamp: Timestamp,
+    previousAnimation: Animation<AnimationObject>
+  ): void => {
+    const toValue = animation.toValue as AffiniteMatrixFlat;
+
+    animation.startMatrices = decomposeMatrixIntoMatricesAndAngles(value);
+    animation.stopMatrices = decomposeMatrixIntoMatricesAndAngles(toValue);
+
+    /** We create an animation copy to animate  single value between 0 and 100
+     **/
```

Comment:
```suggestion
    // We create an animation copy to animate single value between 0 and 100
```

Reply:
I've kept `/** **/` instead of `// `, since I've added more lines to this comment, to explain why we animate between 0 an 100 (instead of 0 and 1), is it okay?

Reply:
Technically that's correct, however we already use `//` for multi-line comments throughout our codebase so maybe let's stick to this convention for the purpose of consistency.